plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestHiveGlueMetastore.java

```diff
@@ -267,12 +267,6 @@ public void cleanupOrphanedDatabases()
         });
     }
 
-    @Override
-    public void testRenameTable()
```

Comment:
hmm I don't think this should be enabled for hive... I only wanted to enable it for delta lake,  I will try to prevent revert to previous behaviour - no rename table with glue for hive